Pascal/Pascal.sublime-syntax

```diff
@@ -47,7 +47,7 @@ contexts:
             0: punctuation.definition.comment.pascal
           push:
```

Comment:
From what I tested `(?!\G)` is not needed as long as we keep using `set` instead of `push`